### ISSUE TITLE
The application was Unable to start correctly (0xc000007b)

### ISSUE THREAD

**Issue by TASINGFTW** [SwareJonge/Dolphin-Lua-Core#3]
When opening Dolphin LUA For the first time, I get this error. 

![90287946-df151800-de46-11ea-9e75-07b8322dd961](https://user-images.githubusercontent.com/69695787/90287981-f05e2480-de46-11ea-8b04-a045e7e78547.png)

I'm assuming it has to do with the .dll files I downloaded to be faulty.

If anybody could help, That would be great!

Thank You!

**Comment by SandPadres** [comment]
I don't know what happened, but I can tell you my process to get Dolphia LUA working. So first, I got Microsoft Studio Build Tools 2017, then I added on VC++ 2015.3 v14.00 (v140) build tool and Windows 10 SDK (10.0.10586). Then I opened up Developer Command Prompt and ran msbuild on Dolphin-Lua-Core/Source/dolphin-emu.sln. I ran into some errors with a file called make_scmrev.h.js, and I fixed the error by adding git.exe to my PATH variable. Then, I ran into some errors with lua.hpp not being found. I installed LuaJIT, and copied the offending files into Dolphin-Lua-Core/Source/Core from LUAJit. Then the compiler was able to finish. This created DolphinD.exe in Dolphin-Lua-Core/Binary/x64. Running DolphinD.exe I ran into an error that lua5.3.dll was missing. I copied lua5.3.dll from another place (Data I think) in Dolphin-Lua-Core to the Dolphin-Lua-Core/Binary/x64 directory, and then it ran.
